pkg/deployments/tasks/scripts/20230116-ta-transition-migrator/input/mainnet.ts

```diff
@@ -0,0 +1,82 @@
+import { flatten } from 'lodash';
+import Task, { TaskMode } from '../../../../src/task';
+import { RoleData } from './types';
+
+const StablePool = new Task('20210624-stable-pool', TaskMode.READ_ONLY, 'mainnet');
+const StablePoolV2 = new Task('20220609-stable-pool-v2', TaskMode.READ_ONLY, 'mainnet');
+const ComposableStablePool = new Task('20220906-composable-stable-pool', TaskMode.READ_ONLY, 'mainnet');
+const ComposableStablePoolV2 = new Task('20221122-composable-stable-pool-v2', TaskMode.READ_ONLY, 'mainnet');
+const WeightedPoolV2 = new Task('20220908-weighted-pool-v2', TaskMode.READ_ONLY, 'mainnet');
+const NoProtocolFeeLbp = new Task('20211202-no-protocol-fee-lbp', TaskMode.READ_ONLY, 'mainnet');
+const ManagedPool = new Task('20221021-managed-pool', TaskMode.READ_ONLY, 'mainnet');
+const AaveRebalancedLinearPool = new Task('20220817-aave-rebalanced-linear-pool', TaskMode.READ_ONLY, 'mainnet');
+const AaveRebalancedLinearPoolV3 = new Task('20221207-aave-rebalanced-linear-pool-v3', TaskMode.READ_ONLY, 'mainnet');
+
+const PoolRecoveryHelper = new Task('20221123-pool-recovery-helper', TaskMode.READ_ONLY, 'mainnet');
+
+const ArbitrumRootGaugeFactoryV2 = new Task('20220823-arbitrum-root-gauge-factory-v2', TaskMode.READ_ONLY, 'mainnet');
+
+const BLABS_OPS_MULTISIG = '0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B';
+const EMERGENCY_SUBDAO_MULTISIG = '0xa29f61256e948f3fb707b4b3b138c5ccb9ef9888';
+const BALLERS_MULTISIG_GAUNTLET = '0xf4a80929163c5179ca042e1b292f5efbbe3d89e6';
+const DAO_MULTISIG = '0x10a19e7ee7d7f8a52822f6817de8ea18204f2e4f';
+
+const createRoleData = (grantee: string, target: string, actionIds: string[]): RoleData[] =>
+  actionIds.map((actionId) => ({ role: actionId, grantee: grantee.toLowerCase(), target: target.toLowerCase() }));
+
+// Hard-coded roles taken from https://forum.balancer.fi/t/bip-131-pool-factory-permission-granting/4144.
+export const roles: RoleData[] = flatten([
```

Comment:
To double check: let's see if we want to migrate every permission that can be found on-chain, or if we want to leave any of these behind.

On top of that, we should wait for https://github.com/balancer-labs/balancer-v2-monorepo/pull/2161 to kick in before deploying this so that those permissions get included in the list as well.

Reply:
Obsolete permissions excluded https://github.com/balancer-labs/balancer-v2-monorepo/pull/2151/commits/204ede1b746e6a04e2018e623c358170513f96ed.

We still need to add the permissions to be granted after executing gauge adder v2 --> v3 migrator (https://github.com/balancer-labs/balancer-v2-monorepo/pull/2162)

Reply:
Pending permissions listed in https://github.com/balancer-labs/balancer-v2-monorepo/pull/2151/commits/8a1b7a3b6886d44be16362e69fb986e7c688c98b.

Remove TODOs after actually triggering the gauge adder permission migration.